### PR TITLE
Modification of the Ethereum-based stablecoin basket to an equally weighted basket of centralized stablecoins: {USDC, USDT}

### PR DESCRIPTION
**Pull request by chicfilabae**
Modification of the Ethereum-based stablecoin basket to an equally weighted basket of centralized stablecoins: {USDC, USDT}. h/t @nicholaspai

### CODE REVIEW

File: UMIPs/umip-30.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The definition of this identifier should be:
 - Identifier name: ELASTIC_STABLESPREAD/USDC
 - Base Currency: ELASTIC_STABLESPREAD
 - Quote Currency: USDC
-- Data Sources: {Uniswap V2: FRAX/ETH} for FRAX, {Uniswap V2: ESD/ETH} for ESD, {Uniswap v2: BAC/ETH} for BAC, {Balancer: MUSD/USDC, Uniswap V2: MUSD/USDC} for MUSD, {Uniswap v2: USDC/ETH} for ETH
+- Data Sources: {Uniswap V2: FRAX/ETH} for FRAX, {Uniswap V2: ESD/ETH} for ESD, {Uniswap v2: BAC/ETH} for BAC, {Bitfinex: UST/USD, Kraken: USDT/USD} for USDT, {Kraken: USDC/USD, Bitstamp: USDC/USD} for USDC, {Uniswap v2: USDC/ETH} for ETH
```
Comment (nicholaspai) [inline UMIPs/umip-30.md:31]: Should the `Bitfinex: UST/USD` be `Bitfinex: USDT/USD`?

Comment (chicfilabae) [reply UMIPs/umip-30.md:31]: Discussed offline, Bitfinex convention in their API is UST

**Review by nicholaspai — APPROVED** [review]
LGTM, I think the new denominator that you propose is more resistant than the relatively illiquid MUSD DEX market. Curious what @smb2796 and @UMAprotocol/eng think

**Comment by smb2796** [comment]
Agree with @nicholaspai's thought. Curious on why TUSD and DAI were excluded though

**Comment by chicfilabae** [comment]
> Agree with @nicholaspai's thought. Curious on why TUSD and DAI were excluded though

Restricted to solely centralized stablecoins issued on Ethereum that were the most liquid and highest ADV in uniform weighting.

**Comment by nicholaspai** [comment]
> > Agree with @nicholaspai's thought. Curious on why TUSD and DAI were excluded though
> 
> Restricted to solely centralized stablecoins issued on Ethereum that were the most liquid and highest ADV in uniform weighting in the minimize economic manipulability

I think that's reasonable given USDC and USDT make up 25/30bn of the stablecoin market and the #3 stablecoin, DAI, is 4x smaller than USDC

**Comment by chicfilabae** [comment]
Agreed

**Pull request merged**